.buildkite/scripts/jdk-matrix-tests/generate-steps.py

```diff
@@ -276,8 +277,8 @@ def integration_tests(self, part: int, parts: int) -> JobRetValues:
             retry=copy.deepcopy(ENABLED_RETRIES),
         )
 
-    def pq_integration_test_parts(self, parts) -> list[JobRetValues]:
-        return list(map(lambda idx: pq_integration_tests(self, idx+1, parts), range(parts))
+    def pq_integration_test_parts(self, parts) -> list[partial[JobRetValues]]:
+        return [partial(self.pq_integration_tests, part=idx+1, parts=parts) for idx in range(parts)]
```

Comment:
Linux tests using this PR: https://buildkite.com/elastic/logstash-linux-jdk-matrix-pipeline/builds/449
Windows tests using this PR: https://buildkite.com/elastic/logstash-windows-jdk-matrix-pipeline/builds/398